tools/SourceKit/lib/SwiftLang/SwiftCompletion.cpp

```diff
@@ -1097,7 +1100,7 @@ static void transformAndForwardResults(
       cargs.push_back(arg.c_str());
     std::string error;
     if (!swiftCodeCompleteImpl(lang, buffer.get(), str.size(), swiftConsumer,
-                               cargs, error)) {
+                               cargs, llvm::vfs::getRealFileSystem(), error)) {
```

Comment:
This should use the provided filesystem, no?  We can stick it in the `session`, if that's the issue.